tests/foreman/cli/test_reporttemplates.py

```diff
@@ -898,23 +898,27 @@ def test_positive_generate_hostpkgcompare(
         }
     )
 
-    hosts_info = []
+    clients = []
     with VMBroker(nick='rhel7', host_classes={'host': ContentHost}, _count=2) as hosts:
         for client in hosts:
             # Create RHEL hosts via broker and register content host
             client.install_katello_ca(default_sat)
             # Register content host, install katello-agent
             client.register_contenthost(local_org['label'], local_ak['name'])
             assert client.subscribed
-            hosts_info.append(Host.info({'name': client.hostname}))
+            clients.append(client)
             client.enable_repo(REPOS['rhst7']['id'])
             client.install_katello_agent()
-        hosts_info.sort(key=lambda host: host['name'])
+        clients.sort(key=lambda client: client.hostname)
```

Comment:
Why the sort? I don't see where it affects the test..

Reply:
So we get the same order as in the report. Otherwise, this can fail e.g. on line 934.